File: js/1132.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
const primeiroNum = Number(lines.shift());
const segundoNum = Number(lines.shift());
if (primeiroNum < segundoNum) {
    somar(primeiroNum, segundoNum);
} else {
    somar(segundoNum, primeiroNum);
}

function somar(menor, maior) {
    let soma = 0;
    for (let i = menor; i <= maior; i++) {
        if (i % 13 !== 0) {
            soma += i;
        }
    }
    return console.log(soma);
}

File: cs/1132.cs
int primeiroNum = int.Parse(Console.ReadLine());
int segundoNum = int.Parse(Console.ReadLine());
int soma = 0;
if (primeiroNum < segundoNum)
{
    for (int i = primeiroNum; i <= segundoNum; i++)
    {
        if (i % 13 != 0)
        {
            soma += i;
        }
    }
    Console.WriteLine(soma);
}
else
{
    for (int i = segundoNum; i <= primeiroNum; i++)
    {
        if (i % 13 != 0)
        {
            soma += i;
        }
    }
    Console.WriteLine(soma);
}